psi4/driver/procrouting/dft_funcs/dict_builder.py

```diff
@@ -140,17 +140,21 @@ def get_functional_aliases(functional_dict):
                 # this ensures that M06-2X-D3, M06-2X-D3ZERO, M062X-D3 or M062X-D3ZERO
                 # all point to the same method (M06-2X-D3ZERO)
                 for alias in functional_aliases:
-                    alias = alias + "-" + dispersion_name.upper()
+                    alias = alias + "-" + dispersion_name.lower()
                     functionals[alias] = func
 
 
-def check_consistency(func_dictionary, name):
+def check_consistency(func_dictionary):
     """
     This checks the consistency of the definitions of exchange and correlation components
     of the functional, including detecting duplicate requests for LibXC params, inconsistent
     requests for HF exchange and missing correlation.
     """
-
+    # 0) make sure method name is set:
+    if "name" not in func_dictionary:
+        raise ValidationError("SCF: No method name was specified in functional dictionary.")
+    else:
+        name = func_dictionary["name"]
```

Comment:
I should be able to trace all the upper/lower and tell, but could you confirm that someone is prevented from defining their own "pBe0" with different alpha?

Reply:
Modified in the latest commit.